about.html

```diff
@@ -12,53 +12,80 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+<header class = 'header'>
+    <img id ='LambdaLOGO' src = 'img/lambda-black.png' alt = 'Lambda is awesome!'>
```

Comment:
😆 I agree Lambda is awesome but your `alt` attributes should be more descriptive of the actual image. So maybe something like `lambda logo`